src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ReportsSubsystemStatus.scala

```diff
@@ -24,15 +25,27 @@ trait ReportsSubsystemStatus extends SprayJsonSupport {
 
   def getStatusFromDropwizardChecks(response: Future[HttpResponse])(implicit ec: ExecutionContext, materializer: Materializer): Future[SubsystemStatus] = {
     response flatMap { resp =>
-      Unmarshal(resp).to[Map[String, DropwizardHealth]].map { dwStatus =>
-        val ok = dwStatus.values.forall(_.healthy)
-        val errors = dwStatus.
+      Unmarshal(resp).to[ConsentStatus].map { status =>
+        val ok = status.ok.get
+        val errors = status.systems.get.
           filter(dw => !dw._2.healthy).
           map(dw => s"Error in ${dw._1}: ${dw._2.message.getOrElse("unspecified error")}").toList
         if (ok)
           SubsystemStatus(ok, None)
         else
           SubsystemStatus(ok, Some(errors))
+      }.recoverWith {
+        case _: Throwable =>
+          Unmarshal(resp).to[Map[String, DropwizardHealth]].map { dwStatus =>
+            val ok = dwStatus.values.forall(_.healthy)
+            val errors = dwStatus.
+              filter(dw => !dw._2.healthy).
+              map(dw => s"Error in ${dw._1}: ${dw._2.message.getOrElse("unspecified error")}").toList
+            if (ok)
+              SubsystemStatus(ok, None)
+            else
+              SubsystemStatus(ok, Some(errors))
+          }
```

Comment:
This is the original logic to parsing consent status that is now used only when the new parsing fails. When https://github.com/DataBiosphere/consent/pull/1235 is merged, the new status format will take effect.